src/OPDSDataAdapter.ts

```diff
@@ -25,13 +26,26 @@ function entryToBook(entry: any, feedUrl: string): BookProps {
       imageUrl = url.resolve(feedUrl, artworkLinks[0].href);
     }
   }
+
+  let detailUrl;
+  let detailLink = entry.links.find(link => link instanceof CompleteEntryLink);
+  if (detailLink) {
+    detailUrl = detailLink.href;
+  }
+
+  // until OPDSParser parses dcterms:publisher...
+  let publisher = entry.unparsed && entry.unparsed["dcterms:publisher"] ? entry.unparsed["dcterms:publisher"][0]["_"] : null;
```

Comment:
this won't work if someone uses a different namespace for dcterms, but we can fix that when we move it to the parser